likecoin/admin/views/metabox.php

```diff
@@ -139,19 +139,37 @@ function likecoin_add_publish_meta_box( $publish_params ) {
 					<?php } ?>
 				</td>
 			</tr>
+			<?php if ( 'Published (testnet)' === $iscn_status['status'] ) { ?>
+				<tr id="likecoin_submit_iscn_testnet">
+					<th><label><?php esc_html_e( 'ISCN (Testnet) Status', LC_PLUGIN_SLUG ); ?></label></th>
+					<td id="lcISCNStatus">
+						<span id="lcISCNPublish" style="display:<?php echo esc_attr( empty( $status['ipfs']['url'] ) ? 'none' : '' ); ?>">
+							<a rel="noopener" target="_blank" href="<?php echo esc_url( $iscn_status['url'] ); ?>">
+								<?php echo 'Published'; ?>
+							</a>
+						</span>
+					</td>
+				</tr>
+			<?php } ?>
 			<tr id="likecoin_submit_iscn">
-				<th><label><?php esc_html_e( 'ISCN (Testnet) Status', LC_PLUGIN_SLUG ); ?></label></th>
+				<th><label><?php esc_html_e( 'ISCN (Main) Status', LC_PLUGIN_SLUG ); ?></label></th>
 				<td id="lcISCNStatus">
-					<?php if ( ! empty( $iscn_hash ) ) { ?>
+					<?php if ( ! empty( $iscn_hash ) && 'Published' === $iscn_status['status'] ) { ?>
```

Comment:
We cannot directly compare the string `Published`, since it is returned by the localization function `__( 'Published', LC_PLUGIN_SLUG )` which might not be English.
Should add a flag in $result about ( `is_dev`) or rename the field (like adding a suffix (`_dev`) to distinguish between testnet and mainnet hash

Reply:
Thanks for pointing out! I added a flag `is_dev_published` to be more precise.